### PR TITLE
Сollections

### PR DESCRIPTION
**Pull request by mikeger**
# Purpose

Initial version of collections. Most of the changes are already reviewed.

Contains

- https://github.com/wireapp/wire-ios/pull/491
- https://github.com/wireapp/wire-ios/pull/489
- https://github.com/wireapp/wire-ios/pull/495

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Collections/CollectionLinkCell.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    }
+}
+
+extension CollectionLinkCell: ArticleViewDelegate {
+    func articleViewWantsToOpenURL(_ articleView: ArticleView, url: URL) {
+        url.open()
+    }
+    
+    func articleViewDidLongPressView(_ articleView: ArticleView) {
+        // TODO: showing menu
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Collections/CollectionLinkCell.swift]: Showing of menu I would add in the subsequent PR

**Comment by typfel** [comment]
💣 conflict

**Comment by typfel** [comment]
Will be be getting some snapshot tests for the collection view controller?

**Comment by mikeger** [comment]
@typfel Yep, coming next!

**Pull request merged**